pages/faqs.tsx

```diff
@@ -0,0 +1,148 @@
+    let groupKey = currentValue[key];
+    if (!acc[groupKey]) {
+      acc[groupKey] = [];
+    }
+    acc[groupKey].push(currentValue);
+    return acc;
+  }, {});
+}
+
+export default function Faqs(props: FAQSPROPS) {
+  const { questionList } = props;
+  const inputRef = useRef<HTMLInputElement>(null);
+  const [questionData, setQuestionData] = useState(questionList);
+
+  const onSearchQuestion = () => {
+    if (!inputRef.current!.value) {
+      return setQuestionData(questionList);
+    }
+
+    const searchResult: FAQSLIST = {};
+    Object.keys(questionList).map((category: string) => {
+      const questionCategory = questionList[category];
+      for (let i = 0; i < questionCategory.length; i++) {
+        const pertanyaan = questionCategory[i].pertanyaan.toLowerCase();
+        const keyword = inputRef.current!.value.toLowerCase();
+        if (pertanyaan.includes(keyword)) {
+          if (!searchResult[category]) {
+            searchResult[category] = [];
+          }
+          searchResult[category].push(questionCategory[i]);
+        }
+      }
+    });
+    setQuestionData(searchResult);
+  };
+
+  return (
+    <Layout>
+      <header>
+        <h1>
+          {/* eslint-disable-next-line @next/next/no-img-element */}
+          <img
+            src="https://firebase-kanvas.imgix.net/warga_bantu_warga/hero_banner.png?auto=format,compress,enhance&fm=pjpg&cs=tinysrgb&fit=scale"
+            alt="Warga Bantu Warga"
+            height="291"
+            width="650"
+          />
+        </h1>
+      </header>
+      <div className="px-2">
+        <div className="max-w-7xl mx-auto py-6">
+          <h2 className="text-3xl font-extrabold text-gray-900">
+            Frequently asked questions
+          </h2>
+          <div className="flex items-center">
+            <input
+              type="text"
+              placeholder="Cari pertanyaan kamu"
+              className="focus:ring-indigo-500 focus:border-indigo-500 block w-full px-2 py-2 sm:text-sm border-gray-300 border-2 rounded-md mt-5 mb-5"
+              ref={inputRef}
+            />
+            <button
+              type="button"
+              className="bg-blue-600 text-white ml-2 py-2 px-6 rounded"
+              onClick={onSearchQuestion}
+            >
+              Cari
+            </button>
+          </div>
+          {Object.keys(questionData).map((category: string) => (
+            <div key={category}>
+              <div className="relative">
+                <div
+                  className="absolute inset-0 flex items-center"
+                  aria-hidden="true"
+                >
+                  <div className="w-full border-t border-gray-300" />
+                </div>
+                <div className="relative flex justify-start">
+                  <span className="pr-3 bg-white text-lg font-medium text-gray-900">
+                    {category}
+                  </span>
+                </div>
```

Comment:
There's a minor styling issue in Safari, please try to fix it, thanks!

<img width="737" alt="Screenshot 2021-07-15 at 19 16 33" src="https://user-images.githubusercontent.com/6315466/125780125-7ce8da87-18e3-4c93-a537-b057e4425655.png">

Reply:
I'm still seeing this issue in [the latest deployment preview](https://60f05cdef9426a00071cec26--wargabantuwarga.netlify.app/faqs).

Can you reproduce this issue in your local while running `yarn dev`?